### PR TITLE
Update the release process

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Please have a look at our [guidelines for contributions](https://github.com/microsoft/appcenter-sdk-android/blob/develop/CONTRIBUTING.md) and consider the following before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] ~Are tests passing locally?~
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests?~
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [ ] ~Did you check UI tests on the sample app? They are not executed on CI.~

## Description

Update the release process from jCenter to Maven.

## Related PRs or issues

[AB#85171](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/85171)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -12,6 +12,7 @@ buildscript {
     repositories {
         google()
         jcenter()
```
Comment (MatkovIvan) [inline build.gradle:14]: Remove it

File: versions.gradle [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 ext {
     versionCode = 62
-    versionName = '4.1.1'
+    versionName = '4.1.1-SNAPSHOT'
```
Comment (MatkovIvan) [inline versions.gradle:10]: How it will be removed for public release?

Comment (AnastasiaKubova) [reply versions.gradle:10]: manually, we discussed it internal

File: scripts/add-snapshot-prefix.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Description: Add a `SNAPSHOT` prefix for `versionName` in `versions.gradle`.
+# Usage: ./scripts/add-snapshot-prefix.sh
+
+# Compute new version
+ gradleVersion=`egrep "versionName = '(.*)'" *.gradle | sed -E "s/^.*versionName = '(.*)'.*$/\1/"`
```
Comment (MatkovIvan) [inline scripts/add-snapshot-prefix.sh:7]:
```suggestion
gradleVersion=`egrep "versionName = '(.*)'" *.gradle | sed -E "s/^.*versionName = '(.*)'.*$/\1/"`
```

File: scripts/add-snapshot-prefix.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+baseGradleVersion=$(echo $gradleVersion| cut -d'-' -f 1)
```
Comment (MatkovIvan) [inline scripts/add-snapshot-prefix.sh:8]:
```suggestion
baseGradleVersion=`echo $gradleVersion | cut -d'-' -f 1`
```

File: scripts/add-snapshot-prefix.sh [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+newVersion=$baseGradleVersion-"SNAPSHOT"
+
+ if [[ "$USE_HASH" == "true" ]]
+ then
```
Comment (MatkovIvan) [inline scripts/add-snapshot-prefix.sh:12]:
```suggestion
 if [ "$USE_HASH" = true ]; then
```

**Review by DmitriyKirakosyan — APPROVED** [review]

**Review by russelarms — APPROVED** [review]

**Pull request merged**